apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -420,22 +420,25 @@ type RolloutStrategy struct {
 	// +optional
 	RollingUpdate *RollingUpdateConfig `json:"rollingUpdate,omitempty"`
 
-	// ApplyStrategy describes how to resolve the conflict if the resource to be placed already exists in the target cluster
-	// and is owned by other appliers.
+	// ApplyStrategy describes when and how to apply the selected resources to the target cluster.
 	// +optional
 	ApplyStrategy *ApplyStrategy `json:"applyStrategy,omitempty"`
 }
 
-// ApplyStrategy describes how to resolve the conflict if the resource to be placed already exists in the target cluster
-// and whether it's allowed to be co-owned by other non-fleet appliers.
+// ApplyStrategy describes when and how to apply the selected resource to the target cluster.
 // Note: If multiple CRPs try to place the same resource with different apply strategy, the later ones will fail with the
 // reason ApplyConflictBetweenPlacements.
 type ApplyStrategy struct {
 	// Type defines the type of strategy to use. Default to ClientSideApply.
-	// Server-side apply is a safer choice. Read more about the differences between server-side apply and client-side
-	// apply: https://kubernetes.io/docs/reference/using-api/server-side-apply/#comparison-with-client-side-apply.
+	// Server-side apply is more powerful and flexible than client-side apply.
+	// You SHOULD use server-side apply to safely resolve any potential drift between the
```

Comment:
could you please define what's the drift? do you mean the drift between current resources in the member and resources to be applied on the hub?

Just curious why cx cares about the drift before the applied.

I thought they would like to know the drift between (current resource in the member + resources to be applied on the hub) vs desired.

Reply:
drift = diff between "previous version in the hub" and "current resource in the member"